package.json

```diff
@@ -13,6 +13,7 @@
     "reshape": "^0.4.0",
     "reshape-loader": "^0.4.0",
     "rooftop-client": "^0.2.1",
+    "spike-util": "^0.4.0",
```

Comment:
This is probably a good example project to base changes off: https://github.com/static-dev/spike-datocms